CrowdStrike/tailwind-toucan-base
# feat: Add error-focus-outline shadow

## 🚀 Description
This PR introduces a new error+focus style

---

## 🔬 How to Test

- Tests should pass
- View https://add-new-error-focus-outline.tailwind-toucan-base.pages.dev
- Scroll down to Shadows and verify you see the new shadow added

---

## 📸 Images/Videos of Functionality

<img width="1258" alt="Screenshot 2023-03-09 at 2 54 50 PM" src="https://user-images.githubusercontent.com/8069555/224141076-ee80a7be-e2a2-4cbd-9f0f-84c047a5b24f.png">
